src/content/zendesk.js

```diff
@@ -39,13 +39,13 @@ setTimeout(() => {
 }, 1000);
 
 
-// Zendesk new UI Apr 2023
+// Zendesk new UI Sept 2023
```

Comment:
Instead of changing directly this code, can you duplicate it and change it, please?

The "Apr 2023" script is still working for me, so I guess that not all users have access to the new changes in the UI introduced in "Sept 2023". 😫 

Therefore this will make it work for all users.